apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -8,51 +8,160 @@ import CourseCatalogBannerBackground from '../../../static/curriculum_catalog/co
+      newFilters[filterName] = [...appliedFilters[filterName], value];
+      console.log(newFilters);
+      setAppliedFilters(newFilters);
+    } else {
+      //Remove unchecked item from applied filters
+      newFilters[filterName] = appliedFilters[filterName].filter(
+        item => item !== value
+      );
+      console.log(newFilters);
+      setAppliedFilters(newFilters);
+    }
+  };
+
+  const handleClear = () => {
+    setAppliedFilters(getEmptyFilters());
+  };
+
+  return (
+    <>
+      <HeaderBanner
+        headingText={i18n.curriculumCatalogHeaderTitle()}
+        subHeadingText={i18n.curriculumCatalogHeaderSubtitle()}
+        short={false}
+        backgroundUrl={CourseCatalogBannerBackground}
+        imageUrl={CourseCatalogIllustration01}
+      />
+      <div className={style.catalogFiltersContainer}>
+        {filterTypes.map(filterType => {
+          return (
+            <FilterCheckboxDropdown
+              key={filterType.name}
+              name={filterType.name}
+              label={filterType.label}
+              allOptions={filterType.options}
+              checkedOptions={appliedFilters[filterType.name]}
+              onChange={e => handleSelect(e, filterType.name)}
+            />
+          );
+        })}
+        <button
+          id="clear-filters"
+          type="button"
+          className={style.catalogClearFiltersButton}
+          onClick={handleClear}
+        >
+          Clear
+        </button>
       </div>
-    </div>
-  </>
-);
+      <div className={style.catalogContentContainer}>
+        <div className={style.catalogContent}>
+          {/*TODO [MEG]: calculate and pass in duration and translated from backend */}
+          {curriculaData
+            .filter(curriculum => !!curriculum.grade_levels)
+            .map(
+              ({
+                key,
+                image,
+                display_name,
+                grade_levels,
+                school_subject,
+                cs_topic,
+              }) => (
+                <CurriculumCatalogCard
+                  key={key}
+                  courseDisplayName={display_name}
+                  imageSrc={image || undefined}
+                  duration={'school_year'} // TODO [MEG] actually pass in this data
+                  gradesArray={grade_levels.split(',')}
+                  subjects={school_subject?.split(',')}
+                  topics={cs_topic?.split(',')}
+                  isTranslated={false} // TODO [MEG]: actually pass in this data
+                  isEnglish={isEnglish}
+                />
+              )
+            )}
+        </div>
+      </div>
+    </>
+  );
+};
 
 CurriculumCatalog.propTypes = {
   curriculaData: PropTypes.arrayOf(curriculumDataShape),
   isEnglish: PropTypes.bool.isRequired,
 };
 
+const FilterCheckboxDropdown = ({
+  name,
+  label,
+  allOptions,
+  checkedOptions,
+  onChange,
+}) => {
+  return (
+    <div id={`${name}-dropdown`} className="dropdown">
```

Comment:
I don't see where these classes (here and below) are defined. I see something in `application.scss`, but it looks like those are for different ids.

Reply:
Oh I was adding in classNames here so it would be easier for Denys to reference these components for styling purposes (the current styles are just placeholders for the work he will do).